### PR TITLE
Issue180

### PR DESCRIPTION
**Pull request by bogdanp05**
Fixed compatibility issues with PostgreSQL

### CODE REVIEW

File: flask_monitoringdashboard/main.py [hunk 1]
```diff
@@ -25,6 +26,16 @@ def endpoint():
         f()
         return 'Ok'
 
+    @app.route('/endpoint2')
```
Comment (mircealungu) [inline flask_monitoringdashboard/main.py]: this does not seem to be related to postgres compatibility... or?

Comment (bogdanp05) [reply flask_monitoringdashboard/main.py]: Of course not:d Just added extra endpoints to test, which I then forgot to remove
